src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -263,6 +263,31 @@
      * or enum declaration (interfaces cannot have static initializers).  The
      * first and only child is a statement list.
      *
+     * <p>For Example:</p>
+     * <pre>
+     * static {
+     *   num = 10;
+     * }
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * --VARIABLE_DEF -&gt; VARIABLE_DEF
```

Comment:
The example does not contain the declaration of the variable `num`. Please either add it to the example, or remove from the tree.

Reply:
Done.